sphinx_gallery/tests/tinybuild/examples/future/plot_future_imports.py

```diff
@@ -1,8 +1,8 @@
 """
-Test __future__ imports across cells
-------------------------------------
+Test ``__future__`` imports across cells
```

Comment:
Question, why did pypandoc complain now but didnt before?

Reply:
Possibly an error or their part, possibly on our partwith the uneascaped underline meaning "underline"